core/src/main/java/org/apache/calcite/plan/volcano/VolcanoRuleMatch.java

```diff
@@ -145,16 +144,16 @@ private String computeDigest() {
       if (i > 0) {
         buf.append(", ");
```

Comment:
as we are changing how the digest is computed anyway, we can simplify this as buf.append(',');

Reply:
done